lib/zendesk_apps_support/validations/requirements.rb

```diff
@@ -184,6 +174,22 @@ def invalid_target_types(requirements)
             end
           end
         end
+
+        def erros(requirements)
```

Comment:
Typo in the method name.
How about renaming it to something like `build_errors` or `compute_errors` to avoid clashing with the `errors` in the `#tap` block?

Reply:
Great catch! 

Both names you suggested are great. I will change it `build_errors`